app/models/manageiq/providers/openstack/cloud_manager/provision/cloning.rb

```diff
@@ -4,7 +4,7 @@ def do_clone_task_check(clone_task_ref)
       instance = openstack.handled_list(:servers).detect { |s| s.id == clone_task_ref }
       status   = instance.state.downcase.to_sym
 
-      return true if status == :active
+      return true if [:active, :error].include?(status)
```

Comment:
So if I understand properly, it would be sufficient to raise the exception right here where the status is checked?